client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
In that case, I would design the string so that correct extraction of the
question, test, & case numbers is trivial with a regex. Just stick extra
text on the end of the `case_id` that's there.

On Mon, Jun 29, 2015 at 4:07 PM, soumyabasu <notifications@github.com>
wrote:

> In client/sources/ok_test/models.py
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/97#discussion_r33524730>
> :
>
> > @@ -100,9 +100,11 @@ def unlock(self, interact):
> >                                  for case in suite.cases])
> >          for suite_num, suite in enumerate(self.suites):
> >              for case_num, case in enumerate(suite.cases):
> > +                case_id = '{} > Suite {} > Case {}'.format(
>
> I think hashing is pointless for this actually, so we should just do a
> string concatenation for this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/97/files#r33524730>.
>